File: desktop/phone_touch/rules/player.py
from phone_touch.types import Button

priority = 1000


def match(window):
    return True


def get_buttons(window):
    return [
        Button('previous-song', 'Previous Song', 'skip-previous', 'XF86AudioPrev'),
        Button('play/pause', 'Play/Pause', 'play-arrow', 'XF86AudioPlay'),
        Button('next-song', 'Next Song', 'skip-next', 'XF86AudioNext'),
    ]

File: desktop-js/src/rules/player.js
import controls, { View, TouchableHighlight, Icon, Text } from '../controls';
import { sendKey } from '../utils';

export default () => (<View key="player-group">
  <Text style={{color: '#ffffff', fontSize: 10}}
        key="player-title">Music</Text>
  <View style={{flexDirection: 'row'}}
        key="player-icons">
    <TouchableHighlight onPress={() => sendKey('XF86AudioPrev')}
                        key="player-prev">
      <Icon style={{color: '#ffffff', fontSize: 60}} name="skip-previous"/>
    </TouchableHighlight>
    <TouchableHighlight onPress={() => sendKey('XF86AudioPlay')}
                        key="player-play">
      <Icon style={{color: '#ffffff', fontSize: 60}} name="play-arrow"/>
    </TouchableHighlight>
    <TouchableHighlight onPress={() => sendKey('XF86AudioNext')}
                        key="player-next">
      <Icon style={{color: '#ffffff', fontSize: 60}} name="skip-next"/>
    </TouchableHighlight>
  </View>
</View>);
